prov/gni/include/gnix.h

```diff
@@ -74,6 +74,10 @@ extern "C" {
 #include "gnix_mr_cache.h"
 #include "gnix_mr_notifier.h"
 
+#if HAVE_UDREG
```

Comment:
Sorry to be picky here, but I think this should be USE_UDREG (we always have it, we just don't always use it).